website/docs/components/stepper/nav/partials/code/how-to-use.md

```diff
@@ -0,0 +1,99 @@
+## How to use this component
+
+The Stepper Nav component is used to track the linear progress of a user in a given on-page flow.
+
+To use this component, you can either pass in the contextual Step and Panel components, or use the `@steps` argument to pass in an array of steps.
+
+### Using contextual components
+
+The `Hds::Stepper::Nav::Step` and `Hds::Stepper::Nav::Panel` are yielded as contextual components. The number of steps and panels added must be equal.
+
+```handlebars
+<Hds::Stepper::Nav as |S|>
+  <S.Step>
+    <:title>Step 1</:title>
+  </S.Step>
+  <S.Step>
+    <:title>Step 2</:title>
+  </S.Step>
+  <S.Step>
+    <:title>Step 3</:title>
+  </S.Step>
+
+  <S.Panel>Content 1</S.Panel>
+  <S.Panel>Content 2</S.Panel>
+  <S.Panel>Content 3</S.Panel>
+</Hds::Stepper::Nav>
+```
+
+### Using the @steps argument
+
+When using the `@steps` argument, use the named block `<:body>` to pass in step content.
+
+```handlebars
+<Hds::Stepper::Nav
+  @steps={{array
+    (hash title="Step 1")
+    (hash title="Step 2")
+    (hash title="Step 3")
+  }}
+>
+  <:body>Content</:body>
```

Comment:
@dchyun Perhaps you could include this example from your component PR in the `:body` here?

```
  <:body>
    {{#if (eq this.currentStep 0)}}
      Step 1 content
    {{else}}
      Step 2 content
    {{/if}}
  </:body>
```